source/fuzz/transformation_add_global_variable.cpp

```diff
@@ -38,6 +39,16 @@ bool TransformationAddGlobalVariable::IsApplicable(
   if (!fuzzerutil::IsFreshId(ir_context, message_.fresh_id())) {
     return false;
   }
+
+  // The storage class must be Private or Workgroup.
+  auto storage_class = static_cast<SpvStorageClass>(message_.storage_class());
+  switch (storage_class) {
+    case SpvStorageClassPrivate:
+    case SpvStorageClassWorkgroup:
+      break;
+    default:
+      return false;
+  }
```

Comment:
Is this a case where we should assert false?

Reply:
Yes, done.